server/lib/models/channel.go

```diff
@@ -65,3 +112,97 @@ func ListChannel(dbif Dbif, req *proto.ChannelListRequest) ([]*proto.Model_Chann
 	}
 	return ret, nil
 }
+
+//takes about 90sec
+func bytesColumnEncode(v []byte) string {
+	result := make([]byte, 3 + 4*len(v))
+	result[0] = 'b'
+	result[1] = '\''
+	start := 2
+	for i := 0; i < len(v); i++ {
+		b := v[i]
+		result[start + 4 * i] = '\\'
+		result[start + 4 * i + 1] = 'x'
+		result[start + 4 * i + 2] = "0123456789abcdef"[b >> 4]
+		result[start + 4 * i + 3] = "0123456789abcdef"[b & 0xf]
+	}
+	result[start + 4 * len(v)] = '\''
+	return string(result)
+}
+func Locales() []string {
+	return []string {"en", "ja", "ko", "zh_Hant", "zh_Hans"}	
+}
+func InsertChannelFixture(dbif Dbif) error {
+	locales := Locales()
+	categories := []string {
+		"Share",
+	    "Ask",
+	    "Commentary",
+	    "Meetup",
+	    "Society",
+	    "Region",
+	    "Culture",
+	    "Science",
+	    "Humanities",
+	    "Food",
+	    "Life",
+	    "Chat",
+	    "Hobby",
+	    "Sports",
+	    "Game",
+	    "Anime",
+	    "Manga",
+	    "Music",
+	    "Health",
+	}
+	idlvs := []proto.Model_Channel_IdentityLevel {
+		proto.Model_Channel_None,
+		proto.Model_Channel_Topic,
+		proto.Model_Channel_Channel,
+		proto.Model_Channel_Account,
+	}
+	dstyles := []proto.Model_Channel_TopicDisplayStyle {
```

Comment:
__[gometalinter]__
* [gotype] undeclared name: proto
* [gotype] dstyles declared but not used